### ISSUE TITLE
Improve the installer/updater to require Touch ID instead of system password

### ISSUE THREAD

**Issue by akmLaVx** [AdguardTeam/AdguardForMac#534]
as requested by user 1403048

> Hello,
> 
> Is there any way you can make the updates for AdGuard on macOS more
> seamless? Would it be possible to prompt for Touch ID instead of the system
> password, or just install updates seamlessly/automatically? AdGuard is one
> of few apps I seem to need to type my password in for every time it updates
> and while it's not the end of the world it's a bit annoying. The pop-up
> "auto updater" app also looks kinda sketchy -- it would be better if that
> actually represented AdGuard better.
> 
> Cheers,
> Michael

**Comment by ameshkov** [comment]
Good idea. I am not sure if this is possible, but we definitely should take a look into it.

**Comment by planet0** [comment]
I'd like to see the auto updater be slightly updated in the meantime, as suggested by Michael - to have the app name changed to 'AdGuard Updater' and a new icon, perhaps having the AdGuard icon present and the current updater icon smaller on the bottom right corner.

**Comment by ameshkov** [comment]
This might require patching Sparkle, not as easy as I expected it to be :(
